implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/DynatraceNamingConventionTest.java

```diff
@@ -30,8 +36,18 @@ void nameStartsWithCustomAndColon() {
         assertThat(convention.name("mymetric", Meter.Type.COUNTER, null)).isEqualTo("custom:mymetric");
     }
 
+    @Test
+    void nameShouldBeSanitized() {
+        assertThat(convention.name("mymetric1._-,", Meter.Type.COUNTER, null)).isEqualTo("custom:mymetric1._-_");
```

Comment:
Is it okay to start with `_` after a dot?

Reply:
@shakuzen No, it will fail actually. As I mentioned in https://github.com/micrometer-metrics/micrometer/issues/1648#issuecomment-546126235, I just focused on handling `system.load.average.1m` as I thought other possible glitches could be handled with renaming them by users. If multiple non-alphabet characters should be handled, another unhappy alphabet prefix like "n" might be necessary. I'm not sure if there's any better solution for this.